Staking Wiki Doc - Reorg + writeup

filippoweb3 opened the pull request:
Added Intro, and overview of NPoS

Commit:
Update learn-staking.md

Added Intro, and overview of NPoS

Commit:
Update learn-staking.md

Edited and moved section about nominator role

Commit:
Updated Intro and NPoS

Added minor edits to the Intro and NPoS responsibilities.

Commit:
Moved and edited accounts

Main edits on the account section. Minor edits to Intro and NPoS

Commit:
Polishing account section

Removed unnecessary info. Some stuff rephrased and some new additions.

Commit:
Reorg staking system overview

Mainly reorganizing content, deleting redundant information, adding links

Commit:
Added simple diagram reward-risk

This is a diagram helping visualize how six different criteria impact on the risk-reward balance when nominating validators

Commit:
Added selection of validators part

Main addition giving some background about selection criteria and some reasoning behind why they are important

Commit:
Minor edits and checks

Prettified, spellcheck, minor edits

Commit:
Consensus definition

Edited definition of consensus and added some hyperlinks

Commit:
Added staking proxies

Added staking proxy accounts; definitions about pallets, calls, extrinsics, etc; minor edits

Commit:
Minor edits

Added Phragmen link, minor edits, prettified

kianenigma reviewed (approved):
Looks overall good, although I only did a quick skim.

Would have been nice to use a better themed graphic.

DrW3RK commented:
Thanks @kianenigma More work is underway! Will tag you once the content in the PR is finalized by the team

filippoweb3 commented:
Hi Kian and Radha, the document revision is not finished. Please do only check "Intro" and "Being a nominator" (excluding the reward distribution part). I just wanted to get a feedback since I am halfway through.

Commit:
PJR and Phragmen

Added PJR concept, Phragmen and link to w3f research

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,321 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+# Introduction
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to add
```
```suggestion
Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to finalize
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,321 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to add
+blocks on the chain. Consensus is the process of agreeing on something, in this case the progression
+of the blockchain or how blocks are added to the chain. Consensus is split into two protocols:
```
Better to show these two as list items

filippoweb3 replied:
Done

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,321 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+blocks on the chain. Consensus is the process of agreeing on something, in this case the progression
+of the blockchain or how blocks are added to the chain. Consensus is split into two protocols:
+`block production`, i.e. the way multiple blocks candidates are produced, and `block finality`, i.e.
+the way only one block out of many candidates is selected and added to the chain. Because in public
```
You can always redirect the reader to another Wiki page that goes in depth on the topic https://wiki.polkadot.network/docs/learn-consensus#probabilistic-vs-provable-finality

```suggestion
the way only one block out of many candidates is selected and added to the canonical chain.
```

filippoweb3 replied:
Added link as requested

Commit:
NPoS overview structure

Removed overview section and restructured NPoS overview

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,321 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+the way only one block out of many candidates is selected and added to the chain. Because in public
+blockchains we have many participants who do not know each other (and probably never will), the
+reach of consensus happens in a secure and trustless way using Proof-of-Work (PoW) or Proof-of-Stake
+(PoS). In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
```
rephrasing

```suggestion
Proof-of-Work (PoW) and Proof-of-Stake(PoS) are well known mechanisms used to reach consensus in a secure and trustless way on public blockchains where we have many participants who do not know each other (and probably never will). In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,321 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+reach of consensus happens in a secure and trustless way using Proof-of-Work (PoW) or Proof-of-Stake
+(PoS). In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
+security of the network is dependent on the amount of capital locked on chain: the more the capital
+locked the lower the chance that someone will be able to successfully attack the network, as they
```
```suggestion
locked the lower the chance of an attack on the network, as the attacker
```

Commit:
Page structure

Changed page structure, minor edits to NPoS overview

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,323 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+(PoS). In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
+security of the network is dependent on the amount of capital locked on chain: the more the capital
+locked the lower the chance that someone will be able to successfully attack the network, as they
+would require a lot of own tokens or to collude different network participants. The process of
```
Anyone who has significant stake in the network would not want to attack it, as it is not in their economic interests. Feel free to rephrase. The PoS network rewards good behaviour and punishes bad behaviour. Any rational actor staking on the network would want to get rewarded. It is not in their economic interests to orchestrate an attack on the network and risk losing their stake.

```suggestion
needs to incur heavy loss to orchestrate a successful attack. The process of
```

filippoweb3 replied:
Text added as requested

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,323 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+security of the network is dependent on the amount of capital locked on chain: the more the capital
+locked the lower the chance that someone will be able to successfully attack the network, as they
+would require a lot of own tokens or to collude different network participants. The process of
+locking tokens on the chain is also called `staking`. This is the main difference between PoS and
```
```suggestion
locking tokens on the chain is called `staking`. 
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,323 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+has been criticized due to the high amount of energy needed for computers to solve such puzzles. In
+PoW networks miners are responsible for adding blocks to the chain, and for doing such work they are
+rewarded with tokens. Similarly to the miners in PoW networks, in PoS networks we have `validators`.
+Token holders can lock funds on chain and for doing so they are getting `staking rewards`. There is
```
Feel free to rephrase

```suggestion
PoW networks miners are responsible for adding blocks to the chain, and for doing such work they are
rewarded with tokens. In order to do that, miners have to compete to solve difficult mathematic puzzles - a solution that has been criticized for the energy wastage. Similar to the 'miners' in PoW networks, PoS networks have `validators` who do not have to compete with each other to solve mathematical puzzles and are instead pre-selected to produce the blocks based on the stake backing them.
Token holders can lock funds on chain and for doing so they are getting `staking rewards`. There is
```

filippoweb3 replied:
Commit accepted and rephrased a bit

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,323 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+DPoS networks weigh validators by stake, oftentimes allowing the highest-stake validators to take
+disproportionate control of the network’s consensus protocol. In DPoS networks, delegators are not
+subject to loss of stake based on the behavior of the validator.
```
Probably better to avoid mention of DPoS right at the beginning. Instead, can highlight what NPoS is and how it differs from other PoS mechanisms like DPoS later on in the doc https://www.gemini.com/cryptopedia/proof-of-stake-delegated-proof-of-stake-consensus-mechanism

I'm leaning towards retaining existing Wiki introduction to NPoS

```suggestion
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements NPoS (Nominated Proof-of-Stake),
a relatively novel and sophisticated mechanism to select the validators who are allowed to participate in 
its [consensus](learn-consensus.md) protocol. The NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
holders to participate as nominators.
```

Commit:
Removed how to stake video

How to stake video removed, added link to support article.

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
Preliminary review implementations

Implemented changes from preliminary review

Commit:
Minor edits

Added / deleted stuff up to the section staking system overview w/o reward part

Commit:
Reorg reward section

Reorganised staking rewards part

Commit:
Slashing & chilling

Major edits to slashing, reorg and minor fixes throughout the doc

Commit:
Changed reward-risk figure

Commit:
Changed layout of reward-risk figure

filippoweb3 commented:
@DrW3RK the PR is ready to go. There might be some minor things but I would like to open a second PR (once this is merged) since this one includes an old version of staking rewards section.

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
-as its [consensus](learn-consensus.md) mechanism. The system encourages 
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holders to participate
-as nominators. Nominators may back up to 
+Here you will lean about what is staking, why is important and how it works for
```
```suggestion
Here you will learn about what staking is, why it is important and how it works on
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+- `block finality`, i.e. the way only one block out of many candidates is selected and added to the canonical chain (see [this](learn-consensus.md/#probabilistic-vs-provable-finality) article for more information about finality).
+
+Proof-of-Work (PoW) and Proof-of-Stake (PoS) are well known mechanisms used to reach consensus in a secure and trustless way on public blockchains, where we have many participants who do not know each other (and probably never will). In PoW networks, miners are responsible for adding blocks to the chain, and for doing such work they are
+rewarded with tokens. Network security relies on the fact that to add blocks miners must compete to solve difficult mathematic puzzles - a solution that has been criticized for the energy wastage. In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
```
```suggestion
typically rewarded with tokens. Network security relies on the fact that the miners must compete to solve difficult mathematic puzzles to add blocks - a solution that has been criticized for the energy wastage. In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+needs to incur heavy loss to orchestrate a successful attack (more on this later on). The process of
+locking tokens on chain is called `staking`. 
+
+Similar to the miners in PoW networks, PoS networks have `validators` who do not have to compete with each other to solve mathematical puzzles and are instead pre-selected to produce the blocks based on the stake backing them.
```
```suggestion
Similar to the miners in PoW networks, PoS networks have `validators` but they do not have to compete with each other to solve mathematical puzzles and are instead pre-selected to produce the blocks based on the stake backing them.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+locking tokens on chain is called `staking`. 
+
+Similar to the miners in PoW networks, PoS networks have `validators` who do not have to compete with each other to solve mathematical puzzles and are instead pre-selected to produce the blocks based on the stake backing them.
+Token holders can lock funds on chain and for doing so they are getting `staking rewards`. There is
```
```suggestion
Token holders can lock funds on chain and for doing so, they are getting `staking rewards`. There is
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+than PoW networks, as participants do not need to have either the technical knowledge about
+blockchain technology nor the experience in running mining equipment. 
+
+PoS ensures that everybody has
```
```suggestion
PoS ensures that everybody participating in the staking process has
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+blockchain technology nor the experience in running mining equipment. 
+
+PoS ensures that everybody has
+"skin in the game" and thus can be held accountable. In case of misbehavior participants to the
```
```suggestion
"skin in the game" and thus can be held accountable. In case of misbehavior participants in the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+PoS ensures that everybody has
+"skin in the game" and thus can be held accountable. In case of misbehavior participants to the
+staking process can be punished or `slashed`, and depending on the gravity of the situation their
```
```suggestion
staking process can be punished or `slashed`, and depending on the gravity of the situation, their
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+PoS ensures that everybody has
+"skin in the game" and thus can be held accountable. In case of misbehavior participants to the
+staking process can be punished or `slashed`, and depending on the gravity of the situation their
+stake can be partly or fully confiscated. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.
```
```suggestion
stake can be partly or fully confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+be allocated. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} uses tools ranging
+from election theory to game theory to discrete optimization, to develop an efficient validator
+selection process that offers fair representation and security, thus avoiding uneven power and
+influence among validators. The `election algorithm` used by {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} bases on the Proportional Justified Representation (PJR) method called [Phragmen](learn-phragmen.md). For more information about PJR methods visit [this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20represen ...
```
Phragmen algorithm is one of the many algorithms that can be used in Polkadot NPoS. We probably have to change the URL for this Wiki doc https://wiki.polkadot.network/docs/learn-phragmen to learn-NPoS and avoid this ambiguity. For more context, check the discussion here https://github.com/w3f/polkadot-wiki/issues/3461

```suggestion
influence among validators. The election algorithms used by {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} are based on the Proportional Justified Representation (PJR) methods like [Phragmen](learn-phragmen.md). For more information about PJR methods visit [this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20representation#) research article.
```

filippoweb3 replied:
@DrW3RK should I open and issue about the URL?

profradha replied:
Sure! We will have to add a redirect for it. It may have already been referenced on multiple third party websites

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+nominators. Both validators and nominators can stake their tokens on chain and receive staking
+rewards at the end of each era. The staking system pays out rewards equally to all validators
+regardless of stake. Thus, having more stake on a validator does not influence the amount of block rewards
+it receives. This avoids centralization of power to few validators. There is a probabilistic
```
```suggestion
it receives. This avoids centralization of power to a few validators. There is a probabilistic
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+[dedicated page](../maintain/maintain-guides-validator-payout.md). Distribution of the rewards are
+pro-rata to all stakers after the validator's commission is deducted.
+
+### Skin in the game
```
Search Engine Friendly subsections. Show up when searching about "staking"

```suggestion
### Skin in the game when Staking
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+The security of PoS networks depends on the amount of staked tokens. This means
+that to successfully attack the network one would need a large amount of tokens that can be accrued
+by one single participant alone or by colluding different participants to act maliciously. In case
```
```suggestion
by one single participant alone or would need different participants to collude and act maliciously. In case
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+that to successfully attack the network one would need a large amount of tokens that can be accrued
+by one single participant alone or by colluding different participants to act maliciously. In case
+of NPoS, if there is an attack both the validator(s) and nominators will be
+`slashed` and in some cases their stake partially or fully confiscated. So there is little interest of acting in a harmful way because all participants can be
```
```suggestion
`slashed` and in some cases their stake partially or fully confiscated by the network and deposited to the treasury. So there is little interest of acting in a harmful way because all participants can be
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+by one single participant alone or by colluding different participants to act maliciously. In case
+of NPoS, if there is an attack both the validator(s) and nominators will be
+`slashed` and in some cases their stake partially or fully confiscated. So there is little interest of acting in a harmful way because all participants can be
+held accountable for bad actions. Also, in NPoS validators are paid equal rewards regardless from
```
```suggestion
held accountable for bad actions. In NPoS, validators are paid equal rewards regardless from
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+of NPoS, if there is an attack both the validator(s) and nominators will be
+`slashed` and in some cases their stake partially or fully confiscated. So there is little interest of acting in a harmful way because all participants can be
+held accountable for bad actions. Also, in NPoS validators are paid equal rewards regardless from
+the amount they have at stake, avoiding thus large payouts to few large validators which would
```
```suggestion
the amount they have at stake, thus avoiding large payouts to few large validators which would
```

DrW3RK reviewed (commented):
Doing the review in batches. So far, very good. Added a few suggestions. Will review the remaining content in 1 or 2 more sessions.

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+Since validator slots are limited, most of those who wish to stake their DOT and contribute to the
+economic security of the network will be nominators, thus here we focus on the role of nominators.
+However, it is worth mentioning that validators do most of the heavy lifting: they run hardware
+equipment and manage
```
Could be on their own hardware or on the cloud

```suggestion
However, it is worth mentioning that validators do most of the heavy lifting: they run the validator nodes and manage
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+a validators. For more information you can take a look at the [nominator guide](learn-nominator.md)
+to understand your responsibilities as a nominator.
+
+If you want to be come a nominator check
```
```suggestion
If you want to become a nominator, check
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+If you want to be come a nominator check
+[this](../maintain/maintain-guides-how-to-nominate-polkadot.md) guide. If you are a beginner and
+would like to securely stake your tokens using Polkadot JS Apps, watch [this](https://support.polkadot.network/support/solutions/articles/65000168057-how-do-i-stake-nominate-on-polkadot-) support article. {{ kusama: The video demonstrates it on Polkadot, but the procedure is the same for Kusama :kusama }}
```
```suggestion
would like to securely stake your tokens using Polkadot JS Apps, refer to [this](https://support.polkadot.network/support/solutions/articles/65000168057-how-do-i-stake-nominate-on-polkadot-) support article. {{ kusama: The tutorial presented in the support article is demonstrated on Polkadot, but the procedure is the same for Kusama :kusama }}
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+### Selection of validators
+
+Choosing validators is not a simple task and it should take into account for nominator reward and
```
```suggestion
The task of choosing validators is not simple, as it should take into account for nominator reward and
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+- amount of era points
+- validator pool, equivalent to own stake (i.e. coming from the validator) + other stake (i.e. coming from nominators)
+- own stake
```
Is this the own stake of the nominator? If so, please mention that explicitly

filippoweb3 replied:
@DrW3RK it is the own stake coming from the validator who nominates himself (specified in brackets). Should I re-phrase?

profradha replied:
Yeah, you can rephrase it something along these lines

- Recent history of the era points earned across eras
- Validator's self stake (Shows skin in the game)
- Total stake backing the Validator  (which is the sum of self stake and the stake coming from nominators)

profradha replied:
Avoid usage of terms like Validator pool (given that pools are now associated with Nomination pools)

filippoweb3 replied:
Changed it as suggested

profradha replied:
If you made those changes locally, commit them to the repo

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+#### In theory
+
+To maximize rewards and minimize risk one could select those validators
```
```suggestion
To maximize rewards and minimize risk, one could select those validators
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+that:
+
+- have era points above average (because they will get more rewards for being active), 
+- have validator pool below average (because they will pay out more rewards per staked {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}), 
```
The word pool can be misleading, especially with the nomination pools being rolled out

```suggestion
- have the total stake backing the validator below the average active validator stake (because they will pay out more rewards per staked {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}), 
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+- have era points above average (because they will get more rewards for being active), 
+- have validator pool below average (because they will pay out more rewards per staked {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}), 
+- have high own stake (because if slashed they have a lot to lose), 
+- have low commission fees but not 0% (because it makes sense that for doing the heavy lifting validators ask for a small commission), 
```
```suggestion
- have low commission fees but not 0% (because it makes sense that for doing the heavy lifting, validators ask for a small commission), 
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+- have validator pool below average (because they will pay out more rewards per staked {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}), 
+- have high own stake (because if slashed they have a lot to lose), 
+- have low commission fees but not 0% (because it makes sense that for doing the heavy lifting validators ask for a small commission), 
+- have on-chain registered identity (because increases trust and you can reach out to them or look at their website or social media), 
```
```suggestion
- have on-chain registered identity (because it adds a layer of trust and possibly provides access to their website and contact details), 
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+:::caution Nominators must periodically check their validators
+
+Nominating is _not_ a "set and forget" operation. The whole NPoS system is dynamic and as such nominators should
```
```suggestion
Nominating is _not_ a "set and forget" operation. The whole NPoS system is dynamic and nominators should
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+Nominating is _not_ a "set and forget" operation. The whole NPoS system is dynamic and as such nominators should
+periodically monitor the performance and reputation of their validators. Failing to do so could
+result in applied slashes and/or rewards not being paid out, even for a prolonged period of time.
```
```suggestion
result in applied slashes and/or rewards not being paid out, possibly for a prolonged period of time.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+:::
+
+Although the theory can be used as a general guideline, the practice is more complicated and
```
```suggestion
Although the theory can be used as a general guideline, in practice it is more complicated and
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+:::
+
+Although the theory can be used as a general guideline, the practice is more complicated and
+following the theory might not necessarily lead to the desired result. Validators might have validator pool below average, low commission and above average era points in one era and then having a totally
```
```suggestion
following the theory might not necessarily lead to the desired result. Validators might have the total stake backing them below average, low commission and above average era points in one era and then having a totally
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+based on identity, no slashes and low commission should be fine. Variability in rewards due to the
+other criteria should level out over time. There might be the case when one validator always gets
+below average era points, this is a case when selecting another validator might help to increase the
+stake performance.
```
Rephrasing this paragraph. If I modified the original intent/ideas presented, feel free to make updates.

```suggestion
different profile in the next one. Selection based the criteria like on-chain identity, slash history and low commission make the staking rewards deterministic. But there are criteria that vary more than others, with era points
being the most variable and thus one of the key probabilistic components of staking rewards. It is not recommended
to change nominations because of the low era points of a validator in a single era. Variability in rewards due to the era points should level out over time. If a validator consistently gets
era points below average, it makes sense to nominate a better performing validator for the health of the network and for increased staking rewards.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+Check
+[this](https://support.polkadot.network/support/solutions/articles/65000150130-how-do-i-know-which-validators-to-choose-)
+support article to understand in detail how to select your own set of validators.
```
```suggestion
Check
[this](https://support.polkadot.network/support/solutions/articles/65000150130-how-do-i-know-which-validators-to-choose-)
support article to understand in detail how to select the set of validators to nominate.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+- **Stash:** This account holds funds bonded for staking, but delegates some functions to a
+  controller account. As a result, you may actively participate to staking with a stash private key kept in
+  a cold wallet like ledger, meaning it stays offline all the time. Stash account keys are used to
+  sign staking actions such as bonding and unbonding funds.
```
You can unbond funds using controller

```suggestion
  a cold wallet like ledger, meaning it stays offline all the time. Stash account keys are used to
  sign staking actions such as bonding additional funds.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+  sign staking actions such as bonding and unbonding funds.
+
+- **Controller:** This account acts on behalf of the stash account, signalling decisions about
+  nominating and validating. It sets preferences like commission (for validators) and payout
```
```suggestion
  nominating and validating. It can set preferences like commission (for validators) and the staking rewards payout
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+  account. The earned rewards can be bonded (locked) immediately for staking on your stash account,
+  which would effectively compound the rewards you receive over time. You could also choose to have
+  them deposited to your controller account or a different account as free (transferable) balance.
+  If you are a validator, it also sets your [session keys](learn-keys.md#session-keys). Controller
```
```suggestion
  If you are a validator, it can also be used to set your [session keys](learn-keys.md#session-keys). Controller
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+  which would effectively compound the rewards you receive over time. You could also choose to have
+  them deposited to your controller account or a different account as free (transferable) balance.
+  If you are a validator, it also sets your [session keys](learn-keys.md#session-keys). Controller
+  accounts only needs enough funds to pay for transaction fees.
```
```suggestion
  accounts only need sufficient funds to pay for the transaction fees.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+:::warning
 
-In staking, you can be either a [nominator or a validator](#validators-and-nominators).
+Never leave high balance on a controller account, such accounts are usually "hot" meaning that the
```
```suggestion
Never leave high balance on a controller account which are usually "hot" as their
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+Never leave high balance on a controller account, such accounts are usually "hot" meaning that the
+private key is stored on the device (PC, phone) and it is exposed to the internet all the time you
+are using it. It is good practice to deposit rewards on the stash account or to send them to another
+cold account.
```
```suggestion
private key is stored on the device (PC, phone) and it is always exposed to the internet for potential hacks and scams. It is good practice to deposit rewards on the stash account or to send them to another
account on a cold wallet.
```

DrW3RK reviewed (commented):
Here is some additional feedback. Will finish the review of the whole doc in the next session.

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
Implemented Radha's feedback

Commit:
Started grammar check using grammarly

DrW3RK committed:
Merge branch 'master' into reorganize-staking-page

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+one can bond funds and nominate some validators. The signature of such
+extrinsic might trigger a reward payout at the end of an era; this is an event inside the chain. This way of having
+transactions categorized within pallets and functionalities makes it possible to create accounts
+having special permissions also called **proxy accounts**.
```
Add link to the Proxy Wiki doc

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+![staking](../assets/NPoS/staking-keys_stash_controller.png)
 
-### 2. Staking System Overview
+This hierarchy of separate key types for stash and controller accounts was designed so that nominators and validator operators can
+protect themselves much better than in systems with only one key. As a rule, the more often one uses
+a private key the higher its visibility and thus the chance it can be stolen. So, if one uses a key
```
```suggestion
This hierarchy of separate keys for stash and controller accounts was designed to add a layer of protection to nominators and validator operators. The more often one exposes and uses a private key, the higher its vulnerability for hacks or scams. So, if one uses a key
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+This hierarchy of separate key types for stash and controller accounts was designed so that nominators and validator operators can
+protect themselves much better than in systems with only one key. As a rule, the more often one uses
+a private key the higher its visibility and thus the chance it can be stolen. So, if one uses a key
+for multiple roles on a blockchain network, security can be easily compromised as the likelihood one
+uses that key often is high. Note that the damage linked to stolen private keys is different
```
```suggestion
for multiple roles on a blockchain network, it is likely that the account can get compromised. Note that the damage linked to stolen private keys is different
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+:::info
+
+For ledger users staking directly on Ledger Live, currently there is only one option to use one
+account as both stash and controller.
```
```suggestion
For ledger users staking directly on Ledger Live, currently there is no option to use separate stash and controller accounts.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+### Staking proxies
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is built using
+[substrate](https://substrate.io/), a modular system to efficiently build blockchains without having necessarily deep knowledge about blockchain technology. Within each module or `pallet` one can `call` different
```
```suggestion
[substrate](https://substrate.io/), a modular system to efficiently build blockchains. Within each module or `pallet` one can `call` different
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+functions that have similar logic. For example, the staking pallet contains all functionalities
+related to staking such as bonding or unbonding funds. The combined information of pallets and calls
+constitutes an `extrinsic`, i.e. a transaction that is executed from outside the chain but that
+triggers an event within the chain. Continuing with the staking example, within the staking pallet
```
```suggestion
triggers an event on the chain. Continuing with the staking example, within the staking pallet
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+related to staking such as bonding or unbonding funds. The combined information of pallets and calls
+constitutes an `extrinsic`, i.e. a transaction that is executed from outside the chain but that
+triggers an event within the chain. Continuing with the staking example, within the staking pallet
+one can bond funds and nominate some validators. The signature of such
+extrinsic might trigger a reward payout at the end of an era; this is an event inside the chain. This way of having
+transactions categorized within pallets and functionalities makes it possible to create accounts
```
Are you sure about bonding funds and nominating validators triggers rewards payouts? I think the idea you are trying to convey here will need rephrasing.

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+transactions categorized within pallets and functionalities makes it possible to create accounts
+having special permissions also called **proxy accounts**.
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
+accounts to which one can delegate signatures to calls from specific pallets. There is thus the
```
```suggestion
In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
accounts which can sign extrinsic calls made to specific pallets on behalf of the proxied account. There is thus the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
+accounts to which one can delegate signatures to calls from specific pallets. There is thus the
+possibility to create staking proxy accounts that can be used to sign only calls from staking,
+session and utility pallets. This makes the stash account even more isolated than using a controller
```
```suggestion
possibility to create staking proxy accounts that can be used to sign only extrinsic calls to staking,
session and utility pallets. This makes the stash account even more isolated than using a controller
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+accounts to which one can delegate signatures to calls from specific pallets. There is thus the
+possibility to create staking proxy accounts that can be used to sign only calls from staking,
+session and utility pallets. This makes the stash account even more isolated than using a controller
+account since now one can bond / unbond / bond more funds using the staking proxy account. However,
```
```suggestion
account since one can bond / unbond / bond more funds using the staking proxy account. However,
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -19,48 +186,84 @@ Nominators may back up to
+account since now one can bond / unbond / bond more funds using the staking proxy account. However,
+it is important to remember that actions on proxy accounts are limited, and in the case of staking
+proxy accounts calls from the balance pallet cannot be signed. This means that it is not possible to
+send funds from a staking proxy. To do that one needs to remove that account as a staking proxy.
```
```suggestion
it is important to remember that actions that can be performed by the proxy accounts are limited, and in the case of staking
proxy, extrinsic calls to the balances pallet cannot be signed. This means it is not possible to do balance transfers
on the proxied account through a staking proxy.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
-objectives of this election mechanism are to maximize the security of the network, and achieve fair
-representation of the nominators. If you want to know more about how NPoS works (e.g. election,
-running time complexity, etc.), please read
+The solution of the election of nominators has to meet certain requirements, such as maximizing the
```
"The solution of the election of nominators" is ambiguous. The election solution comprises of both nominators and validators. It is primarily for electing validators for the active set

```suggestion
The "election solution" which is a connected graph between nominators and validators with the stake as edge weights, has to meet certain requirements, such as maximizing the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
+:::
 
-### 4. Rewards Mechanism
+Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators
```
```suggestion
Validators are paid the same regardless of stake backing them. Validators with less stake will generally pay more to nominators
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
 
-### 4. Rewards Mechanism
+Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators
+per-token than pools with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower staked
```
```suggestion
per-token than the ones with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower staked
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
-### 4. Rewards Mechanism
+Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators
+per-token than pools with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower staked
+validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validator pools will be as evenly distributed as possible, to avoid a concentration of power among a few validators. In the long term, validator pools will have similar levels of stake, with the stake
```
```suggestion
validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validators will be as evenly distributed as possible which avoids concentration of power among a few validators. In the long term, validators will have similar levels of stake, with the stake
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
+Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators
+per-token than pools with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower staked
+validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validator pools will be as evenly distributed as possible, to avoid a concentration of power among a few validators. In the long term, validator pools will have similar levels of stake, with the stake
+being higher for higher reputation validators (meaning that a nominator that is willing to risk more
+by backing a validator with a low reputation will get paid more).
```
```suggestion
being higher for validators with higher reputation. A nominator who is willing to risk more
by backing a validator with a lower reputation will get paid more, provided there are no slashing events.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
-per-token than pools with more stake.
+Before distributing rewards to nominators, validators can create a cut of the reward (a commission) that is not shared with the nominators.
+This cut is a percentage of the block reward, not an absolute value. After the commission gets
+deducted, the remaining portion is based on their staked value and split between the validator and
+all of the nominators who have voted for this validator.
```
```suggestion
deducted, the remaining portion is distributed pro-rata based on their staked value and split between the validator and
all of the nominators whose stake has backed this validator.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
+For example, assume the block reward for a validator is 10 DOT. A validator may specify
+`validator_commission = 50%`, in which case the validator would receive 5 DOT. The remaining 5 DOT
+would then be split between the validator and their nominators based on the proportion of stake each
+nominator had. Note that validators can put up their own stake, and for this calculation, their
+stake acts just as if they were another nominator.
```
```suggestion
nominator had. Note that for this calculation, validator's self-stake acts just as if they were another nominator.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -123,75 +342,48 @@ higher than the minimum active nomination. For more information, check the
+nominators and validator. If a validator's commission is set to 100%, no tokens will be paid out to any
+nominations in the validator pool. Notice in particular that the validator is rewarded twice: once in
+commission fees for validating (if their commission rate is above 0%), and once for nominating
+itself with stake.
```
```suggestion
Thus, a percentage of the reward goes thus to pay the validator's
commission fees and the remainder is paid pro-rata (i.e. proportional to stake) to the
nominators and validator. If a validator's commission is set to 100%, no tokens will be paid out to any
nominations in the validator pool. Notice in particular that the validator is rewarded twice: once in
commission fees for validating (if their commission rate is above 0%), and once for nominating
itself with stake.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -227,6 +419,16 @@ Based on the above rewards distribution, nominators in validator pool B get more
 than those in pool A because pool A has more overall stake. Sam has staked 50 DOT in pool A, but he
 only gets 8.3 in return, whereas Kitty gets 12.5 with the same amount of stake.
```
If you can, try to delete the references to "pools" in the doc and replace them with an appropriate wording

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
 
-We also remark that when the network slashes a validator slot for a misbehavior (e.g. validator
-offline, equivocation, etc.) the slashed amount is a fixed percentage (and NOT a fixed amount),
+Note that the network slashes a validator slot for a misbehavior (e.g. validator
```
```suggestion
Note that the network slashes a validator for a misbehavior (e.g. validator
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
 validators whom they consider to be trustworthy.
 
-The second point to note is that each validator candidate is free to name their desired commission
+Also note that each validator candidate is free to name their desired commission
```
```suggestion
Also, note that each validator candidate is free to name their desired commission
```

Commit:
End run grammarly & fetched on-chain data

- Finished to run grammarly on whole page
- fetched on-chain data of active nominators

Commit:
Merge branch 'reorganize-staking-page' of https://github.com/w3f/polkadot-wiki into reorganize-staking-page

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
-  nominating and validating. It sets preferences like payout account and commission. If you are a
-  validator, it also sets your [session keys](learn-keys.md#session-keys). It only needs enough
-  funds to pay transaction fees.
+In order to be paid your staking rewards, someone must claim them for each validator that you
+nominate. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).
```
> In order to be paid your staking rewards, someone must claim them for each validator that you
nominate.

A bit ambiguous

```suggestion
The distribution of staking rewards to the nominators is not automatic and needs to be triggered by someone. Typically the validators take care of this, but anyone can permissionlessly trigger rewards payout for all the nominators whose stake has backed a specific validator in the active set of that era. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
 
-Since validator slots are limited, most of those who wish to stake their DOT and contribute economic
-security to the network will be nominators.
+:::warning Time limit
```
```suggestion
:::warning Time limit to claim staking rewards
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
-Validators do most of the heavy lifting: they produce new block candidates in BABE, vote and come to
-consensus in GRANDPA, validate the state transition function of parachains, and possibly some other
-responsibilities regarding data availability and [XCM](learn-cross-consensus.md).
+If nobody claims your staking rewards within 84 eras, then you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all their own stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: 28 days :polkadot }}{{ kusama: 7 days :kusama }}, nominators should check if they have pending payouts at least this often.
```
```suggestion
If nobody claims your staking rewards within 84 eras, you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all of their self stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: 28 days :polkadot }}{{ kusama: 7 days :kusama }}, nominators should periodically check if they have pending payouts.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
+:::
 
-![staking](../assets/NPoS/article-2.png)
+Rewards can be directed to the same account used to sign the payout (controller), to the stash account (and either
```
```suggestion
Rewards can be directed to the same account used to sign the payout (controller) or to the stash account (and either
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -360,116 +546,20 @@ particular levels are not implemented or referred to in the code or in the syste
-validator produces no blocks during an epoch and fails to send the heartbeat, it will be reported as
-unresponsive. Slashing may occur depending on the repeated offences and how many other validators
-were unresponsive or offline during the epoch.
+For more information about slashing visit the dedicated page.
```
Link to the Wiki doc

DrW3RK reviewed (commented):
This finishes my review of the doc.

Please run this doc through grammarly after all the suggestions are committed. I request @emresurmeli to make another pass at this doc and read it like a brand new user who is here to learn about Staking system on Polkadot
